guides/_subsections/archive-a-recording.md

```diff
@@ -1,15 +1,15 @@
 ## [Archive a Recording](#archive-a-recording)
-In contrast to <i>Active Recordings</i>, which reside within the container
-of the target application, <i>Archived Recordings</i> reside within persistent
-storage attached to a Cryostat instance. In OpenShift, for example, the
+In contrast to `Active Recordings`, which reside within the container
+of the target application, `Archived Recordings` reside within persistent
+storage attached to a **Cryostat** instance. In **OpenShift**, for example, the
 archives are stored in a `PersistentVolumeClaim`.
 
-Archived recordings are created by performing archival upon active recordings.
-When this is requested, Cryostat connects to the target application and copies
-the Flight Recorder data from the selected active recording into an archived
-recording file in storage. The active source recording may be continuous or
-fixed-duration, may be using any event template, may be in
-any state (`RUNNING`, `STOPPED`, etc.), and may even be a snapshot.
+`Archived recordings` are created by performing archival upon `Active Recordings`.
+When this is requested, **Cryostat** connects to the target application and copies
+the `Flight Recorder` data from the selected `Active Recording` into an `Archived
+Recording` file in storage. The `Active` source recording may be `Continuous` or
+`fixed-duration`, may be using any `event template`, may be in
```

Comment:
probably best to either capitalize both "continuous" and "fixed-duration", or neither

Reply:
also can we make this more concise by changing it to "The active source recording may be continuous or fixed-duration, using any event template, in any state, and may even be a snapshot"